docker-compose.override.yml

```diff
@@ -54,4 +44,3 @@ volumes:
     build:
     critical_css:
     node_modules:
-    vendor:
```

Comment:
If this is to undo the changes made in this PR (https://github.com/elifesciences/journal/pull/1684/files) then we should consider leaving the volumes entry alone.

Reply:
That previous switched `/srv/journal/vendor` from a docker volume (named `vendor`) to a host mount. That made this line unnecessary.

This PR changes the definition of a host mount for consistency, but this `vendor` volume is still not needed.